reel-picker/src/main/java/com/innowisegroup/reelpicker/datetime/LocalTime.kt

```diff
@@ -15,13 +15,35 @@ class LocalTime private constructor(val hour: Int, val minute: Int) : Serializab
 
     override fun hashCode(): Int = 31 * hour + minute
 
+    fun plusHours(hoursToAdd: Int): LocalTime =
+        if (hoursToAdd == 0) {
+            this
+        } else {
+            val newHour = hour + hoursToAdd
+            of(newHour, minute)
+        }
+
+    fun minusHours(hoursToSubtract: Int): LocalTime = plusHours(-hoursToSubtract)
+
+    fun plusMinutes(minutesToAdd: Int): LocalTime =
+        if (minutesToAdd == 0) {
+            this
+        } else {
+            val minutesAfterAdding = hour * MINUTES_PER_HOUR + minute + minutesToAdd
+            val newHour = minutesAfterAdding.floorDiv(MINUTES_PER_HOUR)
+            val newMinute = minutesAfterAdding.mod(MINUTES_PER_HOUR)
+            of(newHour, newMinute)
+        }
```

Comment:
Tests didn't pass with such implementation in case when user tries to add more than 24*60 minutes

Reply:
![image](https://user-images.githubusercontent.com/79689735/186514261-97881668-aa20-4b01-98f9-70074f9c4bc9.png)